spec/helpers/timesheet_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+RSpec.describe TimesheetHelper, type: :helper do
+
+  before(:each) do
+    Timecop.freeze
```

Comment:
Would this test be the same if you didn't freeze time before and after it?

Reply:
It should be the same. This test should always be the same regardless of edgecases like running it near midnight. Removed the timecop before/after.